### PR TITLE
[refs #354] Fixes doc error responsive-spacings

### PR DESCRIPTION
**Pull request by yourownmood**
Pull-request resolving: https://github.com/inuitcss/inuitcss/issues/354

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,8 +3,7 @@
 
 
 ## [Unreleased]
-
-
+- Fix `$inuit-responsive-spacing-directions` documentation in `_utilities.responsive-spacings.scss`. [[#354](https://github.com/inuitcss/inuitcss/issues/354)]
```
Comment (csshugs) [inline CHANGELOG.md]: Can you please put this in a `### Fixes` sub-category:

```
## [Unreleased]

### Fixes
- Fix `$inuit-responsive-spacing-directions` documentation in `_utilities.responsive-spacings.scss`. [[#354](https://github.com/inuitcss/inuitcss/issues/354)]
```

**Comment by yourownmood** [comment]
Done :)

**Review by csshugs — APPROVED** [review]

File: utilities/_utilities.responsive-spacings.scss [hunk 2]
```diff
@@ -9,8 +9,7 @@
  * maps, e.g.:
  *
  *   $inuit-responsive-spacing-directions: (
- *     null: null,
- *     bottom: bottom,
+ *     "-bottom": "-bottom",
```
Comment (csshugs) [inline utilities/_utilities.responsive-spacings.scss]: Why did you remove `null`? Further down in the CSS comment block, [it says](https://github.com/yourownmood/inuitcss/blob/b298de712be7f62a964f46774b85ae16b4596cef/utilities/_utilities.responsive-spacings.scss#L25-L28):

```scss
 * This would bring us the following classes:
 *
 *   .u-margin-small@mobile {}
 *   .u-margin-small@tablet {}
 *   .u-margin-small@desktop {}
 *   .u-margin-small@wide {}
 *   .u-margin-bottom-small@mobile {}
 *   .u-margin-bottom-small@tablet {}
 *   .u-margin-bottom-small@desktop {}
 *   .u-margin-bottom-small@wide {}
```

This is not true anymore when we remove `null` from `$inuit-responsive-spacing-directions`.

**Review by csshugs — CHANGES_REQUESTED** [review]
Please re-add `null` to `$inuit-responsive-spacing-directions` map to showcase possibility for inuitcss users.

File: utilities/_utilities.responsive-spacings.scss (hunk 2 above)
Comment (yourownmood) [reply utilities/_utilities.responsive-spacings.scss]: Correct, null creates the versions without `$inuit-responsive-spacing-sizes`.
Fixed your comment

**Comment by yourownmood** [comment]
Correct, my bad!
null creates the versions without $inuit-responsive-spacing-sizes.

Fixed your comment

File: utilities/_utilities.responsive-spacings.scss [hunk 3]
```diff
@@ -9,8 +9,8 @@
  * maps, e.g.:
  *
  *   $inuit-responsive-spacing-directions: (
- *     null: null,
- *     bottom: bottom,
+      null: null,
```
Comment (csshugs) [inline utilities/_utilities.responsive-spacings.scss]: The leading `*` is missing (Sorry for the nitpicking).

**Review by csshugs — CHANGES_REQUESTED** [review]

File: utilities/_utilities.responsive-spacings.scss (hunk 3 above)
Comment (yourownmood) [reply utilities/_utilities.responsive-spacings.scss]: Done 😄

Comment (csshugs) [reply utilities/_utilities.responsive-spacings.scss]: Thanks 😄

**Review by csshugs — APPROVED** [review]

**Comment by csshugs** [comment]
Well spotted @yourownmood! Thanks so much for taking care of this!
